### DISCUSSION TITLE
Raspberry Pi 3 B+ and the web...

### DISCUSSION THREAD

**Discussion by MrFaul** [servo/servo#32265] [Q&A]
OK I know this is very specific...
but does somebody have a build of servo that can cope with ARM, Ubuntu, Wayland (Sway to be precise) and OpenGL 2.0?
All I want is a working Kiosks browser with hardware acceleration for the raspi 3 B+ and so far nothing works...
Open for any suggestions that run and understands modernish CSS... of course...

**Comment by mukilan** [comment]
We have successfully built Servo for Raspberry Pi 4 & 5, but we haven't attempted building with Pi 3. Servo is also an experimental engine and doesn't support all modern CSS features yet.

>  Open for any suggestions that run and understands modernish CSS... of course...

WPE (https://wpewebkit.org/)  might suit your requirements.

**Reply by MrFaul** [reply]
Thanks appreciate it.
Unfortunately they don't have builds for mantic in the repository.

Was looking for a way that doesn't require me to install a tool chain just for cross compiling one thing...
